src/key.rs

```diff
@@ -31,7 +38,7 @@ pub struct RsaPublicKey {
     /// Typically 0x10001 (65537)
     e: u64,
 
-    #[cfg_attr(feature = "serde", serde(skip))]
+    #[cfg_attr(feature = "serde", serde(skip, default = "default_nparams"))]
```

Comment:
is there no way to construct this during deserialization from the actual values?

Reply:
yeah would need to impl 'De obvs - just needed to get this to compile in the meantime to see errors w/ all-featuers  - will plug this later once I understand more what I'm doing in here :rofl: